schema.sql

```diff
@@ -11,3 +11,58 @@ vet_clinic(# );
 
 ALTER TABLE animals
 vet_clinic-# ADD COLUMN species VARCHAR(255);
+
+
+ CREATE TABLE owners(
+vet_clinic(# id serial PRIMARY KEY,
+vet_clinic(# full_name VARCHAR(255),
+vet_clinic(# age INTEGER
+vet_clinic(# );
+CREATE TABLE
+vet_clinic=# \d
+               List of relations
+ Schema |      Name      |   Type   |  Owner
+--------+----------------+----------+----------
+ public | animals        | table    | postgres
+ public | animals_id_seq | sequence | postgres
+ public | owners         | table    | postgres
+ public | owners_id_seq  | sequence | postgres
+(4 rows)
```

Comment:
- [ ] [OPTIONAL] Please make sure to only copy proper SQL statements as I noticed you're copying the entire text from the `PSQL` CLI which contains unnecessary text that can't be run, so replace the above statement with the below SQL statement as referred by the previous reviewer
```sql
 CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
 );
```